databases/week1/lesson1.sql

```diff
@@ -0,0 +1,38 @@
+SELECT count(id) FROM status;
+
+SELECT count(id) FROM task where due_date is null or 
+ STR_TO_DATE(due_date, '%Y-%m-%d %H:%i:%s') IS NULL;
+
+SELECT *
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+SELECT *
+FROM task
+INNER JOIN status ON task.status_id = status.id
+WHERE status.name != 'Done';
```

Comment:
In SQL we use the keyword NOT when result shouldn't be equal to something. Right syntax is this: 
```
SELECT * FROM <table>
WHERE NOT <column>=<"value">;
```